climpred_skeleton/core.py

```diff
@@ -0,0 +1,25 @@
+import xarray as xr
+
+
+class Verification:
+    """Absolute base class. Begins the pipeline of verifying a forecast.
+
+    Holds a prediction that is being verified against observations. Stores
+    helpful base attributes (`nmember`, etc.) and runs checks on input
+    data.
+    """
+
+    def __init__(self, prediction: xr.Dataset, observation: xr.Dataset) -> None:
+        # Check that inputs are xarray objects, convert to dataset.
+        # Convert to `cftime`.
+        self._prediction = prediction
+        self._observation = observation
+        self._nmember = self._prediction['member'].size
+        self._members = self._prediction['member'].data
+        self._all_verifs = self._observation['time'].data
+        self._all_inits = self._prediction['init'].data
+
+    def _drop_members(self, members_to_remove: list = None):
```

Comment:
Demo of some utility functions that would be appended to the base class. This is used in `m2c` comparison.